src/js/game/hud/parts/waypoints.js

```diff
@@ -304,6 +346,23 @@ export class HUDWaypoints extends BaseHUDPart {
         }
     }
 
+    /**
+     * Sort waypoints by name
+     */
+    sort() {
```

Comment:
I like that refactoring, but could you choose a bit longer name? I.e. `sortWaypoints`. Just trying to avoid builtin method names

Reply:
https://github.com/tobspr/shapez.io/blob/2722366d686fd551a8edceebafc8e78c1e40e976/src/js/game/hud/parts/waypoints.js#L336

I agree